matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/rendezvous/model/RendezvousError.kt

```diff
@@ -14,10 +14,8 @@
  * limitations under the License.
  */
 
-package im.vector.app.features.login.qr
+package org.matrix.android.sdk.api.rendezvous.model
 
-enum class QrCodeLoginErrorType {
-    DEVICE_IS_NOT_SUPPORTED,
-    TIMEOUT,
-    REQUEST_WAS_DENIED,
-}
+import org.matrix.android.sdk.api.rendezvous.RendezvousFailureReason
+
+class RendezvousError(val description: String, val reason: RendezvousFailureReason) : Exception(description)
```

Comment:
You could extend `Failure.FeatureFailure()` instead.
```suggestion
class RendezvousError(val description: String, val reason: RendezvousFailureReason) : Failure.FeatureFailure()
```